shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -275,12 +296,79 @@ bool FlutterWindowsEngine::RunWithEntrypoint(const char* entrypoint) {
               << std::endl;
     return false;
   }
+  FlutterEngineDisplay display;
```

Comment:
Doesn't seem to have an affect beyond teaching devtools to show the correct frame rate in the overlay

Reply:
Same as above, zero out these structs.